src/systems/camera_video_recorder/CameraVideoRecorder.cc

```diff
@@ -379,7 +386,8 @@ void CameraVideoRecorderPrivate::OnPostRender()
           &CameraVideoRecorderPrivate::OnImage, this);
 
       this->videoEncoder.Start(this->recordVideoFormat,
-          this->tmpVideoFilename, width, height);
+          this->tmpVideoFilename, width, height, 25,
```

Comment:
what do you think about adding `fps` as a parameter too ?

Reply:
I'll merge this PR, and add fps this to the `locked_entities_camera_record` PR.